.gitignore

```diff
@@ -38,3 +38,5 @@ libs/*/*/*.a
 
 # Ignore .DS_Store files on OSX
 .DS_Store
+.project
```

Comment:
These .gitignore changes look like they're not needed?

Reply:
See https://github.com/sqlitebrowser/sqlitebrowser/pull/2193#issuecomment-618263402 - some wrong files have accidentally been committed before. The only part one might argue is that the section used there might not be appropriate.

Reply:
No worries. :smile: